.github/workflows/test.yml

```diff
@@ -2,9 +2,19 @@ name: Compact Contracts Test Suite
 
 on:
   pull_request:
+    paths-ignore:
+      - "**.md"
+      - ".gitignore"
+      - "biome.json"
+      - "turbo.json"
```

Comment:
My apprehension with these two is that changes to either can allow undetected failures to sneak into the repo. This isn't a huge deal with biome, but it could be an annoying thing with turbo. If there's a change or a version update only in the json, you will only ever see it if you run it locally or after it's merged indirectly (the next PR that triggers all the checks)

Reply:
If you really think this is worth it, I recommend removing `biome` and `turbo` from the `push` trigger. Worst case and something breaks, we'll catch it after it's merged